auth-web/src/components/auth/manage-business/SearchBusinessNameRequest.vue

```diff
@@ -0,0 +1,144 @@
+            />
+          </template>
+        </v-form>
+        <v-form v-else ref="addNameRequestForm" lazy-validation class="mt-6">
+          <template>
+            <!-- Search for business identifier or name -->
+            <!-- NB: use v-if to re-mount component between instances -->
+            <NameRequestLookup
+              @business="businessName = $event.name; businessIdentifier = $event.identifier"
+            />
+          </template>
+        </v-form>
+      </v-col>
+    </v-row>
+
+    <ManageBusinessDialog
+      :businessIdentifier="businessIdentifier"
+      :businessName="businessName"
+      :dialog="manageBusinessDialog"
+      :isGovStaffAccount="isGovStaffAccount"
+      :userFirstName="userFirstName"
+      :userLastName="userLastName"
+      @add-success="showAddSuccessModal()"
+      @add-failed-invalid-code="showInvalidCodeModal($event)"
+      @add-failed-no-entity="showEntityNotFoundModal()"
+      @add-failed-passcode-claimed="showPasscodeClaimedModal()"
+      @add-unknown-error="showUnknownErrorModal('business')"
+      @business-already-added="showBusinessAlreadyAdded($event)"
+      @on-cancel="cancelAddBusiness()"
+      @on-business-identifier="businessIdentifier = $event"
+    />
+
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from 'vue-property-decorator'
+import { FolioNumberload, LoginPayload } from '@/models/business'
+import BusinessLookup from './BusinessLookup.vue'
+import Certify from './Certify.vue'
+import CommonUtils from '@/util/common-util'
+import HelpDialog from '@/components/auth/common/HelpDialog.vue'
+import { LDFlags } from '@/util/constants'
+import LaunchDarklyService from 'sbc-common-components/src/services/launchdarkly.services'
+import ManageBusinessDialog from './ManageBusinessDialog.vue'
+import NameRequestLookup from './NameRequestLookup.vue'
+import { StatusCodes } from 'http-status-codes'
+import { mapActions } from 'vuex'
+
+@Component({
+  components: {
+    BusinessLookup,
+    Certify,
+    HelpDialog,
+    ManageBusinessDialog,
+    NameRequestLookup
+  },
+  methods: {
+    ...mapActions('business', [
+      'addBusiness',
+      'updateBusinessName',
+      'updateFolioNumber'
+    ])
+  }
+})
+export default class SearchBusinessNameRequest extends Vue {
+  // $refs: {
+  //   addBusinessForm: HTMLFormElement,
+  //   helpDialog: HelpDialog
+  // }
+
+  // @Prop({ default: false }) readonly dialog: boolean
+  @Prop({ default: false }) readonly isGovStaffAccount: boolean
+  @Prop({ default: '' }) readonly userFirstName: string
+  @Prop({ default: '' }) readonly userLastName: string
+
+  // private readonly addBusiness!: (loginPayload: LoginPayload) => any
+  // private readonly updateBusinessName!: (businessNumber: string) => any
+  // private readonly updateFolioNumber!: (folioNumberload: FolioNumberload) => void
+
+  // local variables
+  manageBusinessDialog = { show: false }
```

Comment:
Normally this is a boolean not an object. Are you doing it this way for a particular reason?